src/components/FileDetail/FileDetail.js

```diff
@@ -32,131 +40,155 @@ const FileDetail = ({fileNode, history, match}) => {
-                {fileNode.name}
-              </h3>
-              <div className="md:row-3 lg:cell-10 md:cell-9 cell-12">
-                <GridContainer>
-                  <div className="cell-6 sm:cell-3">
-                    <p className="FileInfo--Title">Status:</p>
-                    <Badge state={sortedVersions[0].node.state} />
-                  </div>
-                  <div className="cell-6 sm:cell-3">
-                    <p className="FileInfo--Title">Document Type:</p>
-                    <div className="flex">
-                      <div className="FileInfo--Icon">
-                        <SvgIcon
-                          kind={fileTypeDetail[fileNode.fileType].icon}
-                          width="24"
-                          height="24"
+            <Grid>
+              <Grid.Row>
+                <Grid.Column>
+                  <Link
+                    to={`/study/${match.params.kfId}/documents`}
+                    data-testid="back-to-filelist"
+                  >
+                    <Button
+                      basic
+                      labeled
+                      icon
+                      size="mini"
+                      labelPosition="left"
+                      floated="left"
+                    >
+                      <Icon name="arrow left" />
+                      Back to All Documents
+                    </Button>
+                  </Link>
+                </Grid.Column>
+              </Grid.Row>
+              <Grid.Row>
+                <Grid.Column>
+                  <Header>
+                    {fileNode.name}
+                    <Button.Group icon labeled size="mini" floated="right">
+                      <Button
+                        icon="pencil"
+                        onClick={() => setDialog('annotation')}
+                        data-testid="edit-button"
+                        content="EDIT"
+                      />
+                      <Button
+                        icon="download"
+                        onClick={e =>
+                          downloadFile(
+                            studyId,
+                            fileNode.kfId,
+                            null,
+                            downloadFileMutation,
+                          )
+                        }
+                        content="DOWNLOAD"
+                      />
+                      <Button
+                        negative
+                        icon="trash alternate"
+                        onClick={e => {
+                          deleteFile({variables: {kfId: fileNode.kfId}});
+                          history.goBack();
+                        }}
+                        content="DELETE"
+                      />
+                    </Button.Group>
+                  </Header>
+                </Grid.Column>
+              </Grid.Row>
+              <Grid.Row stretched>
```

Comment:
I think I'd like to see this section refactored to just using HTML inside of the grid as I don't think it's the quite the correct use of the statistic component